Example/Podfile.lock

```diff
@@ -37,7 +37,7 @@ CHECKOUT OPTIONS:
     :git: https://github.com/square/Paralayout
 
 SPEC CHECKSUMS:
-  AccessibilitySnapshot: cb80a36813f8897825eada35ca5b09b210e472c3
+  AccessibilitySnapshot: 83a064dbe949ad93a8f24f916a558d5ac172b7d1
```

Comment:
Oh, one more thing... ran `pod install` locally again and looks like the checksum changed since this was added.
```suggestion
  AccessibilitySnapshot: 4e1a6b6a16d945a4aca3a7459490fc6e1ccc867e
```